src/DMXSerial2.cpp

```diff
@@ -829,7 +851,95 @@ void DMXSerialClass2::_processRDMMessage(byte CmdClass, uint16_t Parameter, bool
         nackReason = E120_NR_UNSUPPORTED_COMMAND_CLASS;
       }
 
-// ADD: PARAMETER_DESCRIPTION
+    } else if (Parameter == SWAPINT(E120_PARAMETER_DESCRIPTION) && _initData->parametersLength > 0) { // 0x0051
+      if (CmdClass == E120_GET_COMMAND) {
+        if (_rdm.packet.DataLength != 2) {
+          // Unexpected data
+          nackReason = E120_NR_FORMAT_ERROR;
+        } else if (_rdm.packet.SubDev != 0) {
+          // No sub-devices supported
+          nackReason = E120_NR_SUB_DEVICE_OUT_OF_RANGE;
+        } else {
+          uint16_t requestedPid = READINT(_rdm.packet.Data);
+          uint16_t parameterNr;
+          bool8 validPid = false;
+          for (uint16_t n = 0; n < _initData->parametersLength; n++) {
+            if (requestedPid == _initData->parameters[n].pid) {
+              parameterNr = n;
+              validPid = true;
+              break;
+            }
+          }
+          if (!validPid) {
+            // Out of range sensor
+            nackReason = E120_NR_UNKNOWN_PID;
+          } else {
+            _rdm.packet.DataLength = 20 + strnlen(_initData->parameters[parameterNr].description, DMXSERIAL_MAX_RDM_STRING_LENGTH);
+            WRITEINT(_rdm.packet.Data, _initData->parameters[parameterNr].pid);
+            _rdm.packet.Data[2] = _initData->parameters[parameterNr].length;
+            _rdm.packet.Data[3] = _initData->parameters[parameterNr].type;
+            if (_initData->parameters[parameterNr].getSupported && !_initData->parameters[parameterNr].setSupported)
+              _rdm.packet.Data[4] = E120_CC_GET;
+            else if (!_initData->parameters[parameterNr].getSupported && _initData->parameters[parameterNr].setSupported)
+              _rdm.packet.Data[4] = E120_CC_SET;
+            else if (_initData->parameters[parameterNr].getSupported && _initData->parameters[parameterNr].setSupported)
+              _rdm.packet.Data[4] = E120_CC_GET_SET;
```

Comment:
Match the style of the rest of the code:
```suggestion
            if (_initData->parameters[parameterNr].getSupported && !_initData->parameters[parameterNr].setSupported) {
              _rdm.packet.Data[4] = E120_CC_GET;
            } else if (!_initData->parameters[parameterNr].getSupported && _initData->parameters[parameterNr].setSupported) {
              _rdm.packet.Data[4] = E120_CC_SET;
            } else if (_initData->parameters[parameterNr].getSupported && _initData->parameters[parameterNr].setSupported) {
              _rdm.packet.Data[4] = E120_CC_GET_SET;
            }
```